code/modules/reagents/Chemistry-Goon-Toxins.dm

```diff
@@ -156,4 +158,41 @@ datum/reagent/questionmark/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
 	M.adjustToxLoss(1*REM)
 	..()
-	return
\ No newline at end of file
+	return
+
+datum/reagent/itching_powder
+	name = "Itching Powder"
+	id = "itching_powder"
+	description = "Lots of annoying random effects, chances to do BRUTE damage from scratching. 6% chance to decay into 1-3 units of histamine."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.3
+
+/datum/reagent/itching_powder/reaction_mob(var/mob/living/M, var/method=TOUCH, var/volume)
+	if(method == TOUCH)
+		M.reagents.add_reagent("itching_powder", volume)
+		return
+
+datum/reagent/itching_powder/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(prob(rand(5,50)))
+		M << "You scratch at your head."
+		M.adjustBruteLoss(0.2*REM)
+	if(prob(rand(5,50)))
+		M << "You scratch at your leg."
+		M.adjustBruteLoss(0.2*REM)
+	if(prob(rand(5,50)))
+		M << "You scratch at your arm."
+		M.adjustBruteLoss(0.2*REM)
+	if(prob(6))
+		M.reagents.add_reagent("histamine",rand(1,3))
+		M.reagents.remove_reagent("itching_powder",1)
+	..()
+	return
+
+/datum/chemical_reaction/itching_powder
+	name = "Itching Powder"
+	id = "itching_powder"
+	result = "itching_powder"
+	required_reagents = list("fuel" = 1, "ammonia" = 1)
+	result_amount = 2
```

Comment:
May want to add space fungus--it's involved in a number of recipes and having wall fungus in maintenance+a few others areas gives the janitor something to clean.